services/ui-src/src/forms/mcpar/cpc/cpc.schema.ts

```diff
@@ -1,3 +1,23 @@
 import { object } from "yup";
+import {
+  checkbox,
+  date,
+  nested,
+  number,
+  radio,
+  text,
+  url,
+} from "utils/forms/schemas";
 
-export default object({});
+export default object({
+  "cpc-1": text(),
+  "cpc-1-sub": date(),
+  "cpc-2": url(),
+  "cpc-3": radio(),
+  "cpc-3-o5-text": nested(text, "cpc-3", "Other, specify"),
+  "cpc-4a": checkbox(),
+  "cpc-4a-o5-text": nested(text, "cpc-4", "None of the above"),
```

Comment:
Because parent field id has changed to `4a` instead of `4`, we need to change the reference here or it will never show up.
```suggestion
  "cpc-4a-o5-text": nested(text, "cpc-4a", "None of the above"),
```